hazelcast-sql/src/test/java/com/hazelcast/jet/sql/impl/connector/map/SqlPlanCacheTest.java

```diff
@@ -123,6 +130,19 @@ public void test_dmlCaching() {
         assertThat(planCache(instance()).size()).isZero();
     }
 
+    @Ignore // TODO: [sasha] requires Jet parser to be enabled. Uncomment after IMDG removal.
+    @Test
+    public void test_indexCaching() {
```

Comment:
Checking that index is used for `ORDER BY` queries is done elsewhere. Here, we could rather check following scenario:
- create an indexed map
- issue an initial query to cache a plan
- remove the index and create a different one, just slightly different
- verify that plan was invalidated